pyproject.toml

```diff
@@ -62,6 +62,7 @@ twine = ">=4.0.0,<6.0"
 tomlkit = ">=0.12.4,<1.0"
 lazy_loader = ">=0.4"
 reflex-chakra = ">=0.1.1a1"
+coloraide = "^3.3.1"
```

Comment:
I don't want to include another dependency (trying to cut down the default package install size). I think we should make this an optional install for those who want to use the custom color feature. Or reimplement the functions we use, but that seems like a lot of work.

Reply:
I'll look into that.